config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   root 'welcome#index'
 
+  resources :storages
+  post "storages/new"    => "storages#new"
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.